server/liveblog/themes/themes.py

```diff
@@ -81,6 +83,21 @@ def update_registered_theme_with_local_files(self):
                 created.append(theme)
         return (created, updated)
 
+    def on_delete(self, deleted_theme):
+        blogs_service = get_resource_service('blogs')
```

Comment:
yep, I started to work on this issue since I realized the exact thing you are talking about